test/eventrouter.sh

```diff
@@ -60,37 +62,57 @@ else
 
     # Make sure there's no MUX
     # undeploy fluentd
-    stop_fluentd "" $FLUENTD_WAIT_TIME 2>&1 | artifact_out
+    oc label node --all logging-infra-fluentd- 2>&1 | artifact_out
+    os::cmd::try_until_text "oc get daemonset logging-fluentd -o jsonpath='{ .status.numberReady }'" "0" $FLUENTD_WAIT_TIME
     oc set env ds/logging-fluentd MUX_CLIENT_MODE- 2>&1 | artifact_out
-    start_fluentd false 2>&1 | artifact_out
+    oc label node --all logging-infra-fluentd=true 2>&1 | artifact_out
+    os::cmd::try_until_text "oc get pods -l component=fluentd" "^logging-fluentd-.* Running "
 
     warn_nonformatted $essvc '/project.*'
-    warn_nonformatted $esopssvc '/.operations.*'
+    warn_nonformatted $esopssvc '/.operations.*/'
```

Comment:
why would this need the trailing `/` but not `/project.*`?

Reply:
I guess it's not needed.  A leftover in my struggle...  Will remove it.